extensions/interactions/ItemSelectionInput/directives/item-selection-input-validation.service.ts

```diff
@@ -109,6 +110,85 @@ export class ItemSelectionInputValidationService {
     return warningsList;
   }
 
+  private calculateHandledAnswers(
+      ruleInputs: string[],
+      answerChoiceToIndex: Record<string, number>,
+      rule: Rule,
+      handledAnswers: boolean[]) {
+    ruleInputs.forEach((ruleInput) => {
+      var choiceIndex = answerChoiceToIndex[ruleInput];
+      if (rule.type === 'Equals') {
+        handledAnswers[choiceIndex] = true;
+      } else if (rule.type === 'IsProperSubsetOf') {
+        handledAnswers[choiceIndex] = true;
+      } else if (rule.type === 'ContainsAtLeastOneOf') {
+        handledAnswers[choiceIndex] = true;
+      } else if (rule.type ===
+        'DoesNotContainAtLeastOneOf') {
+        for (var i = 0; i < handledAnswers.length; i++) {
```

Comment:
```suggestion
        for (let i = 0; i < handledAnswers.length; i++) {
```
Please use `let` instead of `var`

Reply:
Thanks! Sorry, still getting used to typescript.